README.md

```diff
@@ -23,18 +28,22 @@ High-performance Async REST API, in Python. FastAPI + GINO + Uvicorn + PostgreSQ
 * Create database models to `/app/models/orm`, add them to `/app/models/orm/migrations/env.py` for migrations
 * Create pydantic models in `/app/models/pydantic`
 * Store complex db queries in `/app/models/orm/queries`
+* Store complex tasks in `app/tasks`.
 * Add / edit globals to `/.env`, expose & import them from `/app/settings/globals.py`
+    * Use any coroutine as a background function: store a reference in the `ARQ_BACKGROUND_FUNCTIONS` env.
```

Comment:
personally I don't think `ARQ_BACKGROUND_FUNCTIONS` is a good way of defining worker functions.

Just hard code it, because:
* that will allow you to set other parameters about each worker function
* you'd have to edit your code to add another worker function, and code to call it, so there's no case where you'd want to change this list without changing the code base.

Reply:
kind of makes sense.

I guess this is partly required since arq>=16 doesn't allow different queues.

May be worth creating an issue on arq to at least discuss this, perhaps we could find a reusable way to do something like this?